Dockerfile

```diff
@@ -735,7 +735,7 @@ ARG GOVMOMISHORTCOMMIT="9078b0b"
 ARG GOVMOMIDATE="2023-02-01T04:38:23Z"
 
 USER builder
-WORKDIR ${GOPATH}/src/github.com/vmware/govmomi
+WORKDIR /home/builder/go/src/github.com/vmware/govmomi
```

Comment:
Curious to hear why you prefer the explicit go source path over the one implied by `GOPATH`?

Reply:
`GOPATH` is unset here, so this results in building in `/src/...` under `/`.